index.html

```diff
@@ -25,7 +26,121 @@ <h1 class="header2">I'm Ali<br>Glad to see<br>you!</h1>
                 <li class="github"><a href="https://"><img src="img/Icon -GitHub.png" alt="GitHub"></a></li>
                 <li class="angelist"><a href="https://"><img src="img/Icon - Angelist.png" alt="Angelist"></a></li>
             </ul>
-        <section>
+        <section class="tonic_border">
+        <section class="tonic_project">
+            <a class="tonic_pic" href="https://"><img src="img/tonic_pic.png" alt=""></a>
+            <p class="tonic_name">Tonic</p>
+        </section>
+            <ul class="info1">
+                <li class="canopy">CANOPY</li>
+                <li class="counter1"><img src="img/Counter.png" alt=""></li>
+                <li class="backend">BACK END DEV</li>
+                <li class="counter2"><img src="img/Counter.png" alt=""></li>
+                <li class="year">2015</li>
+            </ul>
+            <p class="description">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+            <ul class="programs">
+                <li class="html"><a href="https://"><img src="img/html.png" alt=""></a></li>
+                <li class="css"><a href="https://"><img src="img/css.png" alt=""></a></li>
+                <li class="javascript"><a href="https://"><img src="img/js.png" alt=""></a></li>
+            </ul>
+            <a class="see_project" href="https://"><img  src="img/see_project.png" alt="see projects"></a>
+        </section>
+        <section class="tonic_border">
+            <section class="tonic_project">
+                <a class="tonic_pic" href="https://"><img src="img/port3.png" alt=""></a>
+                <p class="tonic_name">Multi-Post<br>Stories</p>
+            </section>
+                <ul class="info2">
+                    <li class="canopy">CANOPY</li>
+                    <li class="counter1"><img src="img/Counter.png" alt=""></li>
+                    <li class="backend">BACK END DEV</li>
+                    <li class="counter2"><img src="img/Counter.png" alt=""></li>
+                    <li class="year">2015</li>
+                </ul>
+                <p class="description">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                <ul class="programs">
+                    <li class="html"><a href="https://"><img src="img/html.png" alt=""></a></li>
+                    <li class="css"><a href="https://"><img src="img/css.png" alt=""></a></li>
+                    <li class="javascript"><a href="https://"><img src="img/js.png" alt=""></a></li>
+                </ul>
```

Comment:
- [OPTIONAL] You re-write each of these `<li class="css"><a href="https://"><img src="img/css.png" alt=""></a></li>` without te image inslide the  `a` tag like 
`<li class="css"><a href="https://">JavaScript</a></li>` and use CSS to style it. Whats your take 💭 ?